vllm/attention/backends/flashinfer.py

```diff
@@ -124,7 +125,8 @@ def _get_decode_wrapper(self):
                 self.runner.parallel_config))
             num_kv_heads = self.runner.model_config.get_num_kv_heads(
                 self.runner.parallel_config)
-            use_tensor_cores = num_qo_heads // num_kv_heads > 4
+            use_tensor_cores = (num_qo_heads // num_kv_heads >
+                                4) or envs.VLLM_FLASHINFER_FORCE_TENSOR_CORES
```

Comment:
nit: Switch the order to skip head calculation when the environment variable is set.

```suggestion
            use_tensor_cores = envs.VLLM_FLASHINFER_FORCE_TENSOR_CORES or (num_qo_heads // num_kv_heads > 4)
```